src/components/ProgressArray.tsx

```diff
@@ -8,18 +8,16 @@ import {
 import ProgressCtx from "./../context/Progress";
 import GlobalContext from "./../context/Global";
 import StoriesContext from "./../context/Stories";
+import { timestamp } from "../util/time";
 
 export default () => {
   const [count, setCount] = useState<number>(0);
-  const { currentId, next, videoDuration, pause, bufferAction } = useContext<ProgressContext>(
-    ProgressCtx
-  );
-  const {
-    defaultInterval,
-    onStoryEnd,
-    onStoryStart,
-    onAllStoriesEnd,
-  } = useContext<GlobalCtx>(GlobalContext);
+  const lastTime = useRef<number>(timestamp());
```

Comment:
Have you checked this with something like Next.js? This may not work in server-side rendered apps since it uses `window`.

Reply:
Haven't checked on SSR but changed this in new commit that should resolve this maybe.